### ISSUE TITLE
Keyboard presses result in double or triple keypresses

### ISSUE THREAD

**Issue by ThornedRose** [linuxdeepin/developer-center#292]
I'm not sure what package is causing the issue but since an update, when I first start Deepin Desktop, all keypresses are doubled (sometimes tripled).

Unplugging and replugging the USB keyboard resolved the issue.

At first this also affected the login as well but after some updates to Deepin, it only affected the desktop and the login was resolved.

Now with the latest updates, unplugging and replugging the keyboard does not fix the issue. The keypresses continue to be doubled or tripled.

In the control centre under "Keyboard and Language", repeat delay and rate are both set to the left (short and slow respectively). I have never changed either of these settings since installing Deepin a year ago.

If I change repeat delay from the left to the middle and back again tthhiiss  hhapppeennsss.
It resolves if I remove and replug the keyboard.
Same thing happens if I change the repeat delay. IItt  sssttaarrttsss  ddooiinngg   tthhiiss until I unplug and replug the keyboard.

 OS: Arch Linux 
 Kernel: x86_64 Linux 4.11.3-1-ARCH
Deepin packages that were updated today (and downgraded back to):
deepin-menu (3.1.2-1 -> 3.1.3-1)
deepin-go-lib (1.0.3-1 -> 1.0.4-1)
deepin-dbus-factory (3.1.4-1 -> 3.1.5-1)
deepin-daemon (3.1.9-3 -> 3.1.10-1)
deepin-tool-kit (0.2.9-1 -> 0.2.10-1)
deepin-desktop-schemas (3.1.6-1 -> 3.1.7-1)
deepin-api (3.1.7-1 -> 3.1.8-1)
startdde (3.1.8-1 -> 3.1.9-1)

**Comment by ThornedRose** [comment]
As an experiment, under "Keyboard and Language", I set the repeat delay and rate to the half way mark, back to the left (got double keypresses) and then back to the middle again. Double keypresses stopped. I left them set to the middle and did the upgrades. I checked that they were still set to the middle and restarted.
After restarts, they were still set to the middle and I no longer have double or triple keypresses.

So the problem here seems to be that the keyboard repeat delay and rate were not set to the halfway mark by default. The multiple keypresses is expected behaviour for the repeat delay and rate set to the left BUT they should not have been set to the left having never touched those settings before.
